.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
   "extends": "airbnb/base",
+  "parser": "babel-eslint"
```

Comment:
It seems that `babel-eslint` has slipped into this PR. Can we relocate it to a preceeding PR?